### PR TITLE
PR for Assets and used components styles

### PR DESCRIPTION
**Pull request by kiranpratyush**
This PR if for the assets and component styles that I have used

### CODE REVIEW

File: components/card.css [hunk 1]
```diff
@@ -0,0 +1,68 @@
+.card-imgwrapper{
+    object-fit: cover;
+
+   
+}
+img{
```
Comment (Sen-442b) [inline components/card.css:6]: Suggestion

- Avoid directly styling the element if it's not necessary

**Review by Sen-442b — APPROVED** [review]
Other things look good

File: components/card.css (hunk 1 above)
Comment (kiranpratyush) [reply components/card.css:6]: This is basically a general rule when Using Images I always want that Image should respect the parent's height and width, not its own height and size. So I have used styling on the Img element

**Review by someshtiwarii — APPROVED** [review]
good to go

**Pull request merged**